packages/libs/react-ui/src/components/Divider/Divider.css.ts

```diff
@@ -1,9 +1,15 @@
+import { sprinkles } from '@theme/sprinkles.css';
 import { vars } from '@theme/vars.css';
 import { style } from '@vanilla-extract/css';
 
-export const dividerClass = style({
-  borderColor: `${vars.colors.$borderDefault}`,
-  borderWidth: '1px',
-  margin: `${vars.sizes.$10} 0`,
-  borderBottomWidth: '0',
-});
+export const dividerClass = style([
+  sprinkles({
+    backgroundColor: '$borderDefault',
+    width: '100%',
+    marginY: '$10',
+    border: 'none',
```

Comment:
now this border is not visible anymore

Reply:
it is using backgroundColor and height to show the divider now which was an arbitrary change while moving styles to sprinkles